docs/IE_PLUGIN_DG/AsyncInferRequest.md

```diff
@@ -1,49 +1,66 @@
 # Asynchronous Inference Request {#openvino_docs_ie_plugin_dg_async_infer_request}
 
+@sphinxdirective
+
 Asynchronous Inference Request runs an inference pipeline asynchronously in one or several task executors depending on a device pipeline structure.
-OpenVINO Runtime Plugin API provides the base InferenceEngine::AsyncInferRequestThreadSafeDefault class:
+OpenVINO Runtime Plugin API provides the base :ref:`InferenceEngine::AsyncInferRequestThreadSafeDefault <doxid-class_inference_engine_1_1_async_infer_request_thread_safe_default>` class:
 
-- The class has the `_pipeline` field of `std::vector<std::pair<ITaskExecutor::Ptr, Task> >`, which contains pairs of an executor and executed task.
-- All executors are passed as arguments to a class constructor and they are in the running state and ready to run tasks.
-- The class has the InferenceEngine::AsyncInferRequestThreadSafeDefault::StopAndWait method, which waits for `_pipeline` to finish in a class destructor. The method does not stop task executors and they are still in the running stage, because they belong to the executable network instance and are not destroyed.
+* The class has the ``_pipeline`` field of ``std::vector<std::pair<ITaskExecutor::Ptr, Task> >``, which contains pairs of an executor and executed task.
+* All executors are passed as arguments to a class constructor and they are in the running state and ready to run tasks.
+* The class has the :ref:`InferenceEngine::AsyncInferRequestThreadSafeDefault::StopAndWait <doxid-class_inference_engine_1_1_async_infer_request_thread_safe_default_1ad09f77f1467a13083e5f1a0eba1948b6>` method, which waits for `_pipeline` to finish in a class destructor. The method does not stop task executors and they are still in the running stage, because they belong to the executable network instance and are not destroyed.
```

Comment:
can we avoid such magic string like `doxid-class_inference_engine_1_1_async_infer_request_thread_safe_default_1ad09f77f1467a13083e5f1a0eba1948b6` ? It's totally non maintainable

Reply:
Totally agree but for now we don't have another solution for this, temporary using this